spec/support/book_controller.rb

```diff
@@ -0,0 +1,10 @@
+RSpec.shared_context 'Authenticated User' do
```

Comment:
This modularization is really good, but be careful with the file name...
This should be `shared_contexts.rb` or something like that.